services/catalog/src/simcore_service_catalog/api/routes/services_resources.py

```diff
@@ -41,6 +42,49 @@
 logger = logging.getLogger(__name__)
 
 SIMCORE_SERVICE_COMPOSE_SPEC_LABEL: Final[str] = "simcore.service.compose-spec"
+_DEPRECATED_RESOURCES: Final[list[str]] = ["MPI"]
+_BOOT_MODE_TO_RESOURCE_NAME_MAP: Final[dict[str, str]] = {"MPI": "MPI", "GPU": "VRAM"}
+
+
+def _compute_service_available_boot_modes(
+    settings: list[SimcoreServiceSettingLabelEntry],
+    service_key: ServiceKey,
+    service_version: ServiceVersion,
+) -> list[BootMode]:
+    """returns the service boot-modes.
+    currently this uses the simcore.service.settings labels if available for backwards compatiblity.
+    if MPI is found, then boot mode is set to MPI, if GPU is found then boot mode is set to GPU, else to CPU.
```

Comment:
So you do not need to parse the `image-key` (e.g. for `isolve-mpi`)

Reply:
no. for backward compatibility I still parse the resources label in the usual service labels.